include/deal.II/grid/tria_accessor.h

```diff
@@ -1764,8 +1762,6 @@ class TriaAccessor : public TriaAccessorBase<structdim, dim, spacedim>
   /**
    * Return an object that can be thought of as an array containing all indices
    * from zero to n_faces().
-   *
-   * @note Only implemented for cells (structdim==dim).
```

Comment:
This feature is already implemented to work for all dimensions. So this was probably overlooked.